docs/table_design/automatic_partitioning.md

```diff
@@ -155,7 +155,7 @@ PROPERTIES("replication_num" = "1");
   - When you configure automatic partitioning, you can only use the function `date_trunc` rather than `time_slice`.
   - The syntax for creating multiple partitions all at a time only supports an interval of `1`.
   - After a table that supports automatic partitioning is created, you can use `ALTER TABLE ADD PARTITION` to add partitions for that table. And the statement `ALTER TABLE ADD PARTITION` also has the above limits.
-- Currently, StarRocks's shared-data mode does not support this feature.
+- Since version 3.1, StarRocks's shared-data mode supports the automatic partitioning.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/2d38cb43-0ae2-4f84-b502-ecb0a7045128)
remove "the"